app/models/deck.rb

```diff
@@ -0,0 +1,10 @@
+class Deck < ActiveRecord::Base
+  belongs_to :user
+  validates :title, presence: true
+
+  has_many :cards, dependent: :destroy
+
+  def current? user_id
```

Comment:
Unused method argument - `user_id`. If it's necessary, use `_` or `_user_id` as an argument name to indicate that it won't be used. You can also write as `current?(*)` if you want the method to accept any arguments but don't care about them.<br>Use def with parentheses when there are parameters.